shared/menubar/index.desktop.js

```diff
@@ -151,7 +152,7 @@ class MenubarRender extends React.Component<Props, State> {
       },
       'Divider',
       ...(this.props.loggedIn ? [{title: 'Open main app', onClick: () => this.props.openApp()}] : []),
-      {title: 'Open files', onClick: () => this.props.openApp(Tabs.fsTab)},
+      {title: 'Open folders in finder', onClick: () => this.props.showInFinder('/')},
```

Comment:
Oh wait, this should actually say the appropriate FileUI name per operating system.

Reply:
Cool, let me find out how to resolve that.